Enigmatry.Entry.AspNetCore.SampleApp/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+#pragma warning disable CA2201
+#pragma warning disable IDE0055
+#pragma warning disable CA5394
+#pragma warning disable IDE0052
+
+namespace Enigmatry.Entry.AspNetCore.SampleApp.Controllers;
```

Comment:
A new project contains Sample .NET Core App that we can use for testing of all our tweaking